gobrew_test.go

```diff
@@ -98,6 +98,7 @@ func TestJudgeVersion(t *testing.T) {
 		// 	wantVersion: "1.19.1",
 		// },
 	}
+	os.Unsetenv("GITHUB_TOKEN")
```

Comment:
I think there's no need to remove this variable in the tests.

Reply:
I removed because otherwise the test would fail as my environment had the broken token.

Maybe we can keep this and create a new one that exposes the error using a broken token?

Reply:
I created a separate request in which we get rid of the token

Reply:
So we close this pull req?

To @juev 
Would you please do the merges and make the release by pushing the release tag?

Thanks

Reply:
> So we close this pull req?
> 
> To @juev Would you please do the merges and make the release by pushing the release tag?
> 
> Thanks

ok